src/url.js

```diff
@@ -67,6 +71,21 @@ export function parseUrl(url) {
     search: a.search,
     hash: a.hash,
   };
+
+  // Some IE11 specific polyfills.
+  // 1) IE11 strips out the leading '/' in the pathname.
+  if (info.pathname === '' || info.pathname[0] !== '/') {
```

Comment:
@cramforce we had [tests](https://github.com/ampproject/amphtml/blob/master/test/functional/test-url.js#L50). It wasn't caught because all tests are currently muted for IE11. I tested locally, this PR fixed those failures.

There're other things need to be done before we can un-mute tests for IE11. We can start by introducing "excluded test files" in karma, and then fix them one by one.